src/locales/en/en.json

```diff
@@ -6,6 +6,17 @@
   "ORGANIZATION_LIST_WARNING":"Organization Data Not Found",
   "ORGANIZATION":"Organization",
   "ORGANIZATION_NAME":"Organization Name",
-  "CHANNEL_NAME":"Channel Name",
-  "CHANNEL_ID":"Channel ID"
+  "ORGANIZATION_CHANNEL_NAME":"Channel Name",
+  "ORGANIZATION_CHANNEL_ID":"Channel ID",
+  "USER_FIRST_NAME":"First Name",
+  "USER_LAST_NAME":"Last Name",
+  "USER_EMAIL":"Email",
+  "USER_PHONE":"Phone",
+  "STATUS":"Status",
```

Comment:
add some prefix to all lang constants

- either screen name eg: ORG_DASHBOARD_ / USER_DASHBOARD_STATUS => `ORG_DASHBOARD_STATUS / USER_DASHBOARD_STATUS`
- or entity name eg: USER_ => `USER_STATUS`
- or these are not entity / screen specific eg: COMMON_ =>`COMMON_STATUS`